alchimia/engine.py

```diff
@@ -1,26 +1,58 @@
 from __future__ import absolute_import, division
 
 from sqlalchemy.engine.base import Engine
+from sqlalchemy.exc import StatementError
 
-from twisted.internet.threads import deferToThreadPool
+from twisted._threads import ThreadWorker, AlreadyQuit
```

Comment:
@glyph already addressed this here: https://github.com/alex/alchimia/pull/27#issuecomment-249796069